src/Lykke.Service.Operations/Workflow/Commands/ExecuteOperationCommand.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Lykke.Service.Operations.Workflow.Commands
+{
+    public class ExecuteOperationCommand
```

Comment:
[MessagePackObject] ?
In fact, I think we can just use default MessagePack serialization settings (and set them, if needed).

Reply:
Yes, used messagepack by default in cqrs-module